### PR TITLE
Rm Links to Outdated Methodology Whitepaper

### PR DESCRIPTION
**Pull request by BrettBoval**

### CODE REVIEW

File: src/common/metrics/case_density.tsx [hunk 1]
```diff
@@ -136,19 +136,7 @@ function renderDisclaimer(): React.ReactElement {
       <ExternalLink href="https://ethics.harvard.edu/files/center-for-ethics/files/key_metrics_and_indicators_v4.pdf">
         “Key Metrics for Covid Suppression”
       </ExternalLink>{' '}
-      by Harvard Global Health Institute and others.
-      <br />
-      When estimating the number of people who will become infected in the
-      course of a year, we rely on the{' '}
```
Comment (mikelehen) [inline src/common/metrics/case_density.tsx:142]: We should keep an eye on the inbox report for if we get questions about the estimated infections without this reference anymore.  I'd guess most people don't understand the difference between cases and infections.

File: src/common/metrics/hospitalizations.tsx [hunk 2]
```diff
@@ -158,6 +158,7 @@ function renderDisclaimer(projections: Projections): React.ReactElement {
       <ExternalLink href="/covid-risk-levels-metrics#icu-capacity-used">
         ICU capacity methodology
       </ExternalLink>
+      .
```
Comment (mikelehen) [inline src/common/metrics/hospitalizations.tsx:161]: If you were so inclined, we can probably drop the ICU headroom reference above now. 😁

**Review by mikelehen — APPROVED** [review]
❤️ ❤️ ❤️

File: src/common/metrics/case_density.tsx (hunk 1 above)
Comment (BrettBoval) [reply src/common/metrics/case_density.tsx:142]: 👍

File: src/common/metrics/hospitalizations.tsx (hunk 2 above)
Comment (BrettBoval) [reply src/common/metrics/hospitalizations.tsx:160]: 👍

**Pull request merged**